### ISSUE TITLE
Expose "lights" property to Group and Scene

### ISSUE THREAD

**Issue by MizterB** [balloob/aiohue#6]
Can you please include the lights property in the group class?  Understanding which lights are in a group will be required to correctly map groups to scenes.  This would allow us to address the following issue in HomeAssistant, where the Hue component is not correctly handling duplicate scene names.

https://github.com/home-assistant/home-assistant/issues/13685#issuecomment-385182762

**Comment by hvbhome** [comment]
cant stress my support for this request enough.
This seems not only to be the cause for the hue.hue_activate_scene issue (which needs the groups), but might very well be the cause for other Hue issues started after the same update.

reference here: https://community.home-assistant.io/t/update-to-68-0-1-how-to-debug-config-error-hassio-hue-setup/52224

ever appearing error in the log:

snippet:
```

018-05-02 23:44:19 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.1.212
2018-05-02 23:44:19 WARNING (MainThread) [homeassistant.setup] Setup of hassio is taking over 10 seconds.
2018-05-02 23:44:19 ERROR (MainThread) [homeassistant.core] Timer got out of sync. Resetting
2018-05-02 23:44:20 ERROR (MainThread) [homeassistant.components.hassio.handler] Timeout on /supervisor/ping request
```

among the many timing issues and Hassio not being able to load.

Hope this gets fixed soon, let me know where i could be of any help.

**Comment by MizterB** [comment]
To resolve the HA issue, the light property also needs to be exposed in a Scene.  Pull request addresses both Scene and Group needs.

**Comment by balloob** [comment]
Fixed in #8 and released as 1.5.0

**Comment by MizterB** [comment]
Thank you!  Will update the HA functionality once 1.5.0 is on pypi.
